SoftLayer/CLI/block/detail.py

```diff
@@ -5,16 +5,35 @@
 import SoftLayer
 from SoftLayer.CLI import environment
 from SoftLayer.CLI import formatting
+from SoftLayer.CLI import helpers
 from SoftLayer import utils
 
 
+def get_block_volume_id(volume_id, block_manager):
```

Comment:
lets move this to the block and file managers as `_get_ids_from_username(username)`

Then add it as a resolver to the manager itself.

```python
class BlockStorageManager(utils.IdentifierMixin, object):
    def __init__(self, client):
        self.configuration = {}
        self.client = client
        self.resolvers = [self._get_ids_from_username]
```